web/frontend/src/types/election.ts

```diff
@@ -0,0 +1,76 @@
+import { ID } from './configuration';
+
+export const enum STATUS {
+  // Initial is when the election has just been created
+  Initial = 0,
+  // InitializedNode is when the nodes has been initialized by the dkg service
+  InitializedNodes = 7,
```

Comment:
if this is an internal status for the frontend this is fine. Otherwise the DKG statuses are defined there: https://github.com/dedis/d-voting/blob/065815b355841e48429036e311348e1542263087/services/dkg/mod.go#L20

Reply:
They're are internal status for the frontend but it would probably be better to rename them so that they match with the DKG statuses